task-sdk/docs/conf.py

```diff
@@ -59,7 +67,17 @@
 
 html_theme = "sphinx_airflow_theme"
 html_sidebars = {"**": ["localtoc.html", "globaltoc.html", "searchbox.html", "relations.html"]}
-
+html_title = "Apache Airflow Task SDK Documentation"
+html_short_title = "Task SDK"
+html_favicon = AIRFLOW_FAVICON_PATH.as_posix()
+html_static_path = [SPHINX_DESIGN_STATIC_PATH.as_posix()]
+html_css_files = ["custom.css"]
```

Comment:
So this was the missing link?

Reply:
Yes. The conf references the shared custom.css file from devel-common/sphinx_design/static/

It was not there before

Reply:
This one https://github.com/apache/airflow/blob/main/devel-common/sphinx_design/static/custom.css